include/cutils/properties.h

```diff
@@ -35,8 +35,13 @@ extern "C" {
 ** WARNING: system/bionic/include/sys/system_properties.h also defines
 **          these, but with different names.  (TODO: fix that)
 */
+#ifdef PROP_NAME_MAX
```

Comment:
You already have something similar in the CFLAGS. I would prefer to keep it only there, so as to require minimal changes to the original Android code.